tests/DependencyInjection/ConsoleErrorsExtensionExceptionsTest.php

```diff
@@ -71,49 +71,6 @@ public function testExceptionsDisabled(): void
 		$this->compile();
```

Comment:
> deduplicate testing configuring container parameter using data provider

probably testing of?
\+ the same in next commit

Reply:
Reworded both CMs to `...testing of...`